src/pages/microblocks/[id].vue

```diff
@@ -41,3 +47,11 @@ await useAsyncData(async() => {
   return true
 })
 </script>
+<style scoped>
+.microblock-details-panel {
```

Comment:
Maybe `.microblock-details__microblock-details-panel` is  more precise?

Reply:
yes, fixed. Thanks for the consistency check